WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift

```diff
@@ -167,6 +167,12 @@ protocol ReaderTopicsChipsDelegate: class {
 
         imageLoader.prepareForReuse()
         displayTopics = false
+
+        guard let layout = topicsCollectionView.collectionViewLayout as? ReaderInterestsCollectionViewFlowLayout else {
```

Comment:
What about moving this logic to the `TopicsCollectionView`?

Then we can just call `topicsCollectionView.collapse()` or something like that — and avoid accessing the layout directly.

Reply:
Done in: https://github.com/wordpress-mobile/WordPress-iOS/pull/14682/commits/f1dd20f2968ceeeb8bebf2de092eec1807919982